Collections/ArrayListHashSet.java

```diff
@@ -0,0 +1,18 @@
+import java.util.*;
+
+public class ArrayListHashSet {
+    public static void toHashSet() {
+
+        ArrayList<String> fruits = new ArrayList<>();
+        fruits.addAll(Arrays.asList("Apple", "Pear", "Banana", "Apple", "Cherry", "Fig", "Orange", "Banana", "Apple"));
+        System.out.println("The size of fruit list is: " + fruits.size());
+        HashSet<String> fruitsHashSet = new HashSet<>();
+        for (String element : fruits
+        ) {
+            fruitsHashSet.add(element);

```

Comment:
Don't forget to format code like this
`for (String element : fruits) {
}`

Reply:
I've already got used to format it with combination of ctrl+alt+l. But sometimes it does formatting only with 2nd attempt. Something strange.